pkg/aliases/script/run.go

```diff
@@ -82,7 +82,7 @@ func (script *Script) Run(args []string, opt docker.RunOption) error {
 		// FIXME: timeout get from command line arguments
 		if err := util.Timeout(3*time.Second, fn); err != nil {
 			if _, ok := err.(*util.TimeoutError); ok {
-				_ = command.Process.Kill()
+				_ = syscall.Kill(-command.Process.Pid, syscall.SIGKILL)
```

Comment:
https://github.com/k-kinzal/aliases/pull/53
There was a case that this PR did not fix.